### DISCUSSION TITLE
When will the copr be updated ?

### DISCUSSION THREAD

**Discussion by nicos68** [zellij-org/zellij#2811] [Q&A] [answered]
Hi,

I am running Fedora 38 and I would like to try zellij. However the copr mentioned in the documentation has not been updated since june and is stuck to version 0.37, which is unfortunate because it lacks the session manager. Does someone know if the copr will be updated soon ? I do see more up to date forks of the copr being created but I’d much prefer using something at least _somewhat_ endorsed by the project rather than some copr created by someone I don’t know. I guess I’ll just install it using rust for now…

**Comment by imsnif** [comment, marked as answer]
Hey - just to be very clear: The third party repositories are not even "somewhat" endorsed by the project. We do not monitor them and have no idea what goes on there. :)

**Reply by nicos68** [reply]
Okay, that's what I was suspecting. It's a bit surprising because in the doc it is said that "The easiest way to install Zellij is through a package for your OS", but that's fine.

**Reply by imsnif** [reply]
It is indeed the easiest! We also, as the doc mentions, provide them for convenience (==easiness). But we do not, as the doc also mentions, take responsibility for them or their contents.
